src/MizarData/include/MizarData/SamplingWithFrameTrackComparisonReport.h

```diff
@@ -85,55 +86,66 @@ class SamplingCounts {
-      absl::flat_hash_map<SFID, CorrectedComparisonResult> fid_to_corrected_comparison_results)
+  explicit SamplingWithFrameTrackComparisonReportTmpl(
+      Baseline<Counts> baseline_sampling_counts,
+      Baseline<FrameTrackStats> baseline_frame_track_stats,
+      Comparison<Counts> comparison_sampling_counts,
+      Comparison<FrameTrackStats> comparison_frame_track_stats,
+      absl::flat_hash_map<SFID, CorrectedComparisonResult> fid_to_corrected_comparison_results,
+      const absl::flat_hash_map<SFID, std::string>* sfid_to_names)
       : baseline_sampling_counts_(std::move(baseline_sampling_counts)),
         baseline_frame_track_stats_(std::move(baseline_frame_track_stats)),
         comparison_sampling_counts_(std::move(comparison_sampling_counts)),
         comparison_frame_track_stats_(std::move(comparison_frame_track_stats)),
-        fid_to_corrected_comparison_results_(std::move(fid_to_corrected_comparison_results)) {}
+        fid_to_corrected_comparison_results_(std::move(fid_to_corrected_comparison_results)),
+        sfid_to_names_(sfid_to_names) {}
 
-  [[nodiscard]] const Baseline<SamplingCounts>& GetBaselineSamplingCounts() const {
+  [[nodiscard]] const Baseline<Counts>& GetBaselineSamplingCounts() const {
     return baseline_sampling_counts_;
   }
-  [[nodiscard]] const Comparison<SamplingCounts>& GetComparisonSamplingCounts() const {
+  [[nodiscard]] const Comparison<Counts>& GetComparisonSamplingCounts() const {
     return comparison_sampling_counts_;
   }
 
-  [[nodiscard]] const Baseline<orbit_client_data::ScopeStats>& GetBaselineFrameTrackStats() const {
+  [[nodiscard]] const Baseline<FrameTrackStats>& GetBaselineFrameTrackStats() const {
     return baseline_frame_track_stats_;
   }
-  [[nodiscard]] const Comparison<orbit_client_data::ScopeStats>& GetComparisonFrameTrackStats()
-      const {
+  [[nodiscard]] const Comparison<FrameTrackStats>& GetComparisonFrameTrackStats() const {
     return comparison_frame_track_stats_;
   }
 
   [[nodiscard]] const CorrectedComparisonResult& GetComparisonResult(SFID sfid) const {
     return fid_to_corrected_comparison_results_.at(sfid);
   }
 
+  [[nodiscard]] const absl::flat_hash_map<SFID, std::string>& GetSfidToNames() const {
+    return *sfid_to_names_;
+  }
+
  private:
-  Baseline<SamplingCounts> baseline_sampling_counts_;
-  Baseline<orbit_client_data::ScopeStats> baseline_frame_track_stats_;
+  Baseline<Counts> baseline_sampling_counts_;
+  Baseline<FrameTrackStats> baseline_frame_track_stats_;
 
-  Comparison<SamplingCounts> comparison_sampling_counts_;
-  Comparison<orbit_client_data::ScopeStats> comparison_frame_track_stats_;
+  Comparison<Counts> comparison_sampling_counts_;
+  Comparison<FrameTrackStats> comparison_frame_track_stats_;
 
   absl::flat_hash_map<SFID, CorrectedComparisonResult> fid_to_corrected_comparison_results_;
+
+  const absl::flat_hash_map<SFID, std::string>* sfid_to_names_;
 };
 
+using SamplingWithFrameTrackComparisonReport =
```

Comment:
As per [style guide](https://google.github.io/styleguide/cppguide.html#Aliases) for aliases in the public API:

>When defining a public alias, document the intent of the new name, including whether it is guaranteed to always be the same as the type it's currently aliased to, or whether a more limited compatibility is intended.